File: Modul 5/Python/PRAK501-2310817310005-MuhammadNurwahyudiAdhitama.py
def max_bilangan(num1, num2, num3, num4):
    max_val = num1

    if num2 > max_val:
        max_val = num2
    if num3 > max_val:
        max_val = num3
    if num4 > max_val:
        max_val = num4

    return max_val

a, b, c, d = map(int, input("").split())
hasil = max_bilangan(a, b, c, d)
print(hasil)

File: Modul 5/C/PRAK501-2310817310005-MuhammadNurwahyudiAdhitama.c
#include <stdio.h>
int MaxBilangan(int num1, int num2, int num3, int num4){
    int max = num1;

    if (num2 > max){
        max = num2;
    }
    if (num3 > max){
        max = num3;
    }
    if (num4 > max){
        max = num4;
    }
    return max;
}
int main() {
int a, b, c, d; 
scanf("%d %d %d %d", &a, &b, &c, &d);
int hasil = MaxBilangan(a, b, c, d);
printf("%d", hasil); 
return 0;
}

